nixos/modules/services/networking/shairport-sync.nix

```diff
@@ -27,36 +24,34 @@ in
         '';
       };
 
-      package = lib.options.mkPackageOption pkgs "shairport-sync" { };
```

Comment:
Same

Reply:
this was due to the conflicts. github shows just the removal. Due to rebase this should be fixed.